test/base-traits.jl

```diff
@@ -31,7 +31,7 @@ if VERSION < v"0.5.0-dev"
     @test istrait(IsCallable{Base.AddFun})
 end
 
-if VERSION >= v"0.6.0-dev"
-    @test istrait(IsIterable{Array})
-    @test !istrait(IsIterable{Cmd})
-end
+@test istrait(IsIterable{Array})
+@test !istrait(IsIterable{Cmd})
+@test method_exists(start, Tuple{Base.UnitRange{Int}})
+@test method_exists(start, Tuple{Base.UnitRange})
```

Comment:
Sorry, I wasn't clear at all, I meant:
```
@test istrait(IsIterable{Base.UnitRange{Int}})
@test istrait(IsIterable{Base.LogicalIndex})
```
and not testing the method_exists directly.